components/Containers/NavBar/NavItem/index.module.css

```
@@ -4,7 +4,12 @@
     gap-2
     rounded
     px-3
-    py-2;
+    py-2
+    duration-75;
```

```suggestion
    motion-safe:duration-75;
```
It would be better to manage animations according to system preferences for users who may be sensitive. 

- https://drafts.csswg.org/mediaqueries-5/#prefers-reduced-motion
- https://tailwindcss.com/docs/animation#prefers-reduced-motion